src/Documenter.jl

```diff
@@ -160,6 +160,12 @@ For example if you are using GitLab.com, you could use
 makedocs(repo = \"https://gitlab.com/user/project/blob/{commit}{path}#{line}\")
 ```
 
+or for AzureDevOps:
+
+```julia
+makedocs(repo = \"https://dev.azure.com/org/project/_git/repo?path={path}&version={commit}{line}&lineStartColumn=1&lineEndColumn=1\")
+```
```

Comment:
Perhaps we could merge this into the previous code block, and use comments to indicate which hosting service it applies to? Just to keep the docstring a bit shorter.

Reply:
Yeah good idea